Gemfile

```diff
@@ -43,6 +43,9 @@ gem 'config'
 gem 'honeybadger'
 gem 'view_component'
 
+# DLSS/domain-specific dependencies
+gem 'cocina-models'
```

Comment:
I might lobby for us to kick around how we want to organize our `Gemfile`s to make sure we want to switch from alphabetical to alphabetical-by-domain or whatever. Do we have preferences, @sul-dlss/infrastructure-team ❓ 

I have a slight preference for straight up alphabetical but I'm open to other suggestions.

Reply:
I'm OK with any consistent approach.

Reply:
Rather, any consistent approach that is in a separate PR.